### ISSUE TITLE
One-finger cursor move does not work if started from left or right side of touchpad

### ISSUE THREAD

**Issue by bodqhrohro** [iberianpig/xSwipe#24]
Configuration of xSwipe seems does not control this behavior. Side-scrolling is disabled in xorg.conf (and AFAIK, it affects only the right side). The width of "dead" start areas is quite large (about 1/5 of whole touchpad width). Additionally I noticed dragging is sometimes self-activated for unknown reason when started move, not double-tapped drag. I use plain X.Org (i.e. no DEs' settings-daemons) on Debian 9.

**Comment by y023rus** [comment]
I have the same issue on Mint 17 Mate.

**Comment by Snd18** [comment]
I have a DELL XPS 13 and the same issue, I thought my touchpad don't work well (I didn't try this without xSwipe).
This issue is on Ubuntu 12.04 and Ubuntu 14.04 as far I tried it.

**Comment by iberianpig** [comment]
It is by design to avoid unnatural movement of pointer. 
Pointer also moves each time two fingers edge swipe if it allows to move pointer.

**Comment by bodqhrohro** [comment]
So maybe disable this behavior if no one edge-swipe rule is set?

**Comment by iberianpig** [comment]
No, the behavior of pointer  is not affected  by config of 1 finger swipe.

**Comment by jat255** [comment]
I'll also note that this is a very undesirable behaviour on my machine (dell latitude e7240). The left and right "edges" are approximately 30% (each) of my touchpad, so in order to move the pointer around, I can only use the very middle of my touchpad. Is there any way to disable this "feature"?
